src/pages/about.js

```diff
@@ -63,7 +63,7 @@ export default function Home() {
         <div className="container py-6">
           <div className="row mb-5">
             <div className="col-md-10 offset-md-1">
-              <h2 className="text-center">
+              <h2 className="text-center" id='team'>
```

Comment:
Is this in anticipation of some link to this section? Otherwise I don't think adding this has any effect.